src/browser/components/RequestsProcessor/TestInvoke/TestInvoke.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
-import { string, func } from 'prop-types';
+import { any, objectOf, func } from 'prop-types';
 
 export default class TestInvoke extends React.Component {
   static propTypes = {
-    script: string.isRequired,
+    result: objectOf(any).isRequired,
```

Comment:
Should we make a shape for this? @mhuggins

Reply:
Possibly. As per [this tutorial](http://cityofzion.io/neon-js/docs/en/basic_createscript.html#simple-calls-with-createscript) it would resemble something like:
```
shape({
  script: string.isRequired,
  state: string.isRequired,
  gas_consumed: string.isRequired,
  stack: arrayOf(any).isRequired
});
```

and we can even further breakdown the stack array, since it has this shape:
```
shape({
  type: string.isRequired,
  value: string.isRequired
});
```

I'll put those in.

Reply:
Shapes added @DalderupMaurice